applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -379,17 +351,18 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
     /// <param name="max_pipe_height"> maximum allowed pipe height</param>
     /// <param name="PipeElements"> vector of all pipe elements</param>
     /// <returns>tuple of grow bool and number of open pipe elements</returns>
-    std::tuple<bool, int> check_status_tip_element(unsigned int          n_open_elements,
-                                                   unsigned int          n_elements,
-                                                   double                max_pipe_height,
-                                                   std::vector<Element*> PipeElements)
+    template <typename FilteredElementType>
+    std::tuple<bool, int> check_status_tip_element(unsigned int               n_open_elements,
+                                                   unsigned int               n_elements,
+                                                   double                     max_pipe_height,
+                                                   const FilteredElementType& PipeElements)
```

Comment:
to adhere to the style guide
```suggestion
    std::tuple<bool, int> check_status_tip_element(unsigned int               NOpenElements,
                                                   unsigned int               NElements,
                                                   double                     MaxPipeHeight,
                                                   const FilteredElementType& rPipeElements)
```

Reply:
done